HEADER

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
```

Comment:
we shoudl have one of these files already in the repo

Reply:
Agree that it should exist, but it appears not to, which is why I created it.

Reply:
https://github.com/paritytech/polkadot/blob/master/file_header.txt